editor/js/Editor.js

```diff
@@ -619,3 +623,40 @@ Editor.prototype = {
 	}
 
 };
+
+// RectAreaLightHelper was designed to work only if it is a child of its light.
+// Since that cannot be done in the edtior, the updateMatrixWorld function is overwritten to apply the matrix calculations to the light.
+THREE.RectAreaLightHelper.prototype.updateMatrixWorld = function () {
```

Comment:
> RectAreaLightHelper was designed to work only if it is a child of its light.

@WestLangley Do you know the reasons for this? I've checked the history of `RectAreaLightHelper` but found nothing.

Other users have already run into issues because of how `RectAreaLightHelper` works (see https://discourse.threejs.org/t/rectarealight-remaining-at-0-0-0/10057). I think it would be less confusing if adding the helper to the light is not required (similar to e.g. `PointLightHelper`). This modification here would not be necessary then.

Reply:
>Do you know the reasons for this? I've checked the history of RectAreaLightHelper but found nothing.

#14935

I expect all light helpers will be changed to match the behavior of `RectAreaLightHelper` once lights are controlled by the quaternion.

Reply:
As long as we try to keep the helpers consistent, I'm fine with it. Thanks for the link 👍